doc/source/whatsnew/v0.23.1.txt

```diff
@@ -81,7 +81,7 @@ Indexing
 I/O
 ^^^
 
--
+- bug in :meth:`nested_to_record` when :meth:`json_normalize` was called with None values in nested levels  (:issue:`21158`)
```

Comment:
nested_to_record is not public, don't use it here

use double backticks around None

say nested levels in JSON